spaces/S000182/properties/P000087.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000182
+property: P000087
+value: true
+refs:
+  - mathse: 2726886
+    name: Show that the product topology on $X\times Y$ is the same as the disjoint union topology on $\bigsqcup_{y\in Y}X$.
+---
+
+Let $I$ denote {S3}. By {{mathse:2726886}}, 
+$$X = \bigsqcup_{i \in I} \mathbb{Q} \approx I \times \mathbb{Q}.$$
+Since {S3|P87}, {S27|P87}, and a product of topological groups is a topological group, it follows that $X$ has a group topology.
```

Comment:
```suggestion
Let $D$ denote {S3} and $\mathbb Q$ denote {S27}. By {{mathse:2726886}}, 
$$X = \bigsqcup_{d \in D} \mathbb{Q} \cong D \times \mathbb{Q}.$$
Since {S3|P87} and {S27|P87}, and a product of topological groups is a topological group, it follows that $X$ {P87}.
```

Reply:
![image](https://github.com/user-attachments/assets/0cecca77-e4ac-40bc-aeb5-f314b0d0251c)